docs/reference/operators.rst

```diff
@@ -122,6 +122,12 @@ not visible *region*
 --------------------
 The part of the given region which is *not* visible from the ego object (as above, based on the ego's :term:`visible region`).
 
+.. _{region} visible from ({Point} | {OrientedPoint}):
```

Comment:
This made me realize we don't have a "R not visible from P" operator. We should add it for symmetry.

Reply:
Added!